README.md

```diff
@@ -1,46 +1,103 @@
 
 ## How to use
 
-Before building, you will need to fulfill the following dependencies
+Assuming you are running Ubuntu/Debian clean on your Orange Pi RK3588/3566 ([Orange Pi 5 series](https://orangepi.vn/tu-khoa-san-pham/opi5series)/ [Orange Pi 3B](https://orangepi.net/product-tag/orange-pi-3b))
 
-* xtensor
-* spdlog
-* libfmt
-* piper-phoenomize
-* onnxruntime (1.14 or 1.15)
-* A C++20 capable compiler
+1. You first need to instal rknpu lib, the fastest way is using Petrolus ezrknn installer
 
-(API/Web server)
-* Drogon
-* libsoxr
-* libopusenc
-    * You'll need to build this from source if on Ubuntu 22.04. Package available starting on 23.04
+```bash
+https://github.com/Pelochus/ezrknn-toolkit2
+cd ezrknn-toolkit2
+sudo bash install.sh
+```
 
-(RKNN support)
-* [rknnrt >= 1.6.0](https://github.com/rockchip-linux/rknn-toolkit2/tree/v1.6.0/rknpu2/runtime/Linux/librknn_api)
+2. then you need to install git-lfs to clone models (which is large size)
+```bash
+(. /etc/lsb-release && curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo env os=ubuntu dist="${DISTRIB_CODENAME}" bash)
+sudo apt-get install git-lfs
+```
 
-In which `piper-phoenomize` and `onnxruntime` binary (not the source! Unless you want to build yourselves!) likely needs to be downloaded and decompressed manually. Afterwards run CMake and point to the folders you recompressed them.
+3. Install some dependencies
+```bash
+sudo apt install -y cmake build-essential
+sudo apt install -y libxtensor-dev nlohmann-json3-dev libspdlog-dev libopus-dev libfmt-dev libjsoncpp-dev
+sudo apt install -y espeak-ng libespeak-ng-dev libogg-dev libsoxr-dev
+```
 
+4. Install drogon
 ```bash
+cd ~
+sudo apt install libssl-dev pkg-config libbotan-2-dev libc-ares-dev uuid-dev doxygen
+git clone https://github.com/drogonframework/drogon
+cd drogon
+git submodule update --init --recursive
 mkdir build
 cd build
-cmake .. -DORT_ROOT=/path/to/your/onnxruntime-linux-aarch64-1.14.1 -DPIPER_PHONEMIZE_ROOT=/path/to/your/piper-phonemize-2023-11-14 -DCMAKE_BUILD_TYPE=Release
-make -j
-# IMPORTANT! Copy espeak-ng-data or pass `--espeak_data` CLI flag
-cp -r /path/to/your/piper-phonemize-2023-11-14/share/espeak-ng-data .
+cmake ..
+make -j4
+sudo make install
 ```
 
-Afterwards run `paroli-cli` and type into the console to synthesize speech. Please refer to later sections for generating the models.
+5. Install libopusenc
+```bash
+cd ~
+wget https://archive.mozilla.org/pub/opus/libopusenc-0.2.1.tar.gz
+tar -xvzf libopusenc-0.2.1.tar.gz
+cd libopusenc-0.2.1
+./configure
+make -j4
+sudo make install
+```
+this part is optional, some time the ldconfig not update your /usr/local/lib, then you need to do it manually
+`sudo nano /etc/ld.so.conf.d/local.conf`
+add this line `/usr/local/lib` then save and exit
+
+7. Cloning models
+```bash
+cd ~
+https://huggingface.co/thanhtantran/piper-paroli-rknn-model
```

Comment:
missing `git clone`?

Also I looked into the models you uploaded. They have the same SHA256 as mine. MIT license requires attribution.